index.html

```diff
@@ -274,6 +274,31 @@ <h3>Languages</h3>
       
        </section>
      </main>
+      <div class="contact__container">
+ 
+          <section class="contact__container_me">
+              
+            <form action="https://formspree.io/f/xbjevbkv" method="POST">
+              <div class="contact__info">
+              <h2 class="Contact__me" >Contact me</h2>
+              <p class="Contact__text" >If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                I’d love to help with it!</p>
+              <label for="contact__name">
+                <input type="text" id="name" name="name" placeholder="Enter your name..." required maxlength="30">
+              </label>
+              <label for="email">
+                <input type="email" id="message" name="email" placeholder="Enter your email..." required/>
+              </label>
+              <label for="message-box">
+                <textarea id="message-box" name="message" placeholder="Please enter your message here" required maxlength="500"></textarea>
+              </label>
+              <button type="submit" class="contact__button" name="submit">
+                <p>Get in touch</p>
+              </button>
```

Comment:
:red_circle: [REQUIRED] Please make sure the button style matches the Figma Design

|FIgma|Your design|
|:-------:|:--------------:|
|![figma](https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/26736582/587dba37-a666-4a88-8e6c-d7d7b98b92b2)|![yako](https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/26736582/27cecd27-f457-4d8e-874c-c6bda2db7af2)|